doc/migration.md

```diff
@@ -1,18 +1,140 @@
- There is a dedicated task to migrate data from prior versions of CTIA. 
- This task will run through all configured stores, transform and copy data to new elasticsearch indices. 
- A migration state will be stored in a configured state to enable restart.
+There is a dedicated task to migrate data from prior versions of CTIA. 
+This task will run through all configured stores, transform and copy data to new Elasticsearch indices. This is possible to migrate these data in another cluster.
```

Comment:
Last sentence could be clearer. Perhaps `It is possible to migrate data between clusters.`.